src/modules/accommodations/dto/request/search-accommodation-params.dto.ts

```diff
@@ -0,0 +1,34 @@
+import { IsDate, IsOptional, IsString, Min, IsInt, Max } from 'class-validator';
+
+export class SearchAccommodationParamsDto {
+  @IsString()
+  @IsOptional()
+  search: string;
+
+  @IsString()
+  @IsOptional()
+  category: string;
+
+  @IsString()
+  @IsOptional()
+  location: string;
```

Comment:
why do you need location field. this field can come in search field. User can write either location or name of property. Remove this field